### DISCUSSION TITLE
I am getting this error while pushing the code.

### DISCUSSION THREAD

**Discussion by sumitvajarinkar** [smv1999/CompetitiveProgrammingQuestionBank#556] [Q&A]
![Screenshot (106)](https://user-images.githubusercontent.com/60393530/125932682-a0c90752-fca1-4575-82c0-5472decb0dd3.png)

**Comment by smv1999** [comment]
CD to the repo folder and then execute the push command. 

That is, you are inside Arrays folder now. Move back to CompetitiveProgrammingQuestionBank by executing :
cd ..
Then execute the push command.

**Comment by sumitvajarinkar** [comment]
I tried cd.. but again same error.
![Screenshot (107)](https://user-images.githubusercontent.com/60393530/125983153-6984808c-7d5e-48f7-919b-ff365c6d2633.png)

**Comment by smv1999** [comment]
Are you sure you cloned the forked repository ? 

Also, you can find some information [here](https://github.com/smv1999/CompetitiveProgrammingQuestionBank/blob/master/CONTRIBUTING.md).

**Reply by sumitvajarinkar** [reply]
Yes, I successfully did trapping rainwater but after that, I'm getting errors for others.
Now, I'm again doing from start fork -> clone -> and further process.

**Reply by sumitvajarinkar** [reply]
Same problem while pushing again.

**Reply by sumitvajarinkar** [reply]
I drag it in the forked repo and merged finally. ✌
